### PR TITLE
Added aliases to webpack

### PR DESCRIPTION
**Pull request by r3nya**
Hey!
We can use aliases from webpack to resolve paths.

### CODE REVIEW

**Comment by pgilad** [comment]
Isn't it safer to be more explicit with aliases:
instead of `actions` use `^actions\/`

**Comment by r3nya** [comment]
@pgilad good catch! :+1:

File: webpack.config.js [hunk 1]
```diff
@@ -38,7 +38,16 @@ module.exports = {
     ],
   },
   resolve: {
-    extensions: ['', '.js', '.jsx']
+    extensions: ['', '.js', '.jsx'],
```
Comment (aziz) [inline webpack.config.js:41]: I'm doing all this with just this single line

```
root: [path.resolve(__dirname, 'src'), path.resolve(__dirname, 'node_modules')],
```

But I'm not sure wether it's better or not.

**Comment by tj** [comment]
neat! Do you know if there's a way we can just do imports relative to root without aliases? ex `/components/foo`, that would be a tiny bit more elegant I think if possible, instead of having an alias per dir

**Comment by aziz** [comment]
@tj this is how you can do it 

``` js
  resolve: {
    extensions: ['', '.js', '.jsx'],
    root: [
       path.resolve(__dirname, 'client'), 
       path.resolve(__dirname, 'node_modules')
    ],
  },
```

and the you do imports like this: 

``` js
import Header from 'components/Header'
```

**Comment by tj** [comment]
cool having the ones that don't need to be maintained sounds good to me!

**Comment by tj** [comment]
Still like this but we'll have redo it with the above changes

**Pull request closed without merging**